Bottom: 8507fefbd46b0e060c3dd0d9ea106c6955e64b47
Top:    1b1244eb1dd3b0f77bb25b732fc7497967ade11e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2015-03-23 00:10:47 +0000

Put back layer opacity to its old behaviour

The exact details of this setting aren't right yet,
so revert them so I can build a release.


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 58f8757..286a362 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1732,7 +1732,7 @@ ghid_draw_everything (BoxType *drawn_area)
   }
 
   /* draw all layers in layerstack order */
-#define FADE_FACTOR 0.8
+#define FADE_FACTOR 1
   number_phys_on_top = max_phys_group - min_phys_group;
   for (i = ngroups - 1; i >= 0; i--) {
     bool is_this_physical = drawn_groups[i] >= min_phys_group &&
